Lib/test/test_float.py

```diff
@@ -754,6 +754,18 @@ def test_format(self):
         self.assertEqual(format(INF, 'f'), 'inf')
         self.assertEqual(format(INF, 'F'), 'INF')
 
+        # underscores
+        x = 123_456.123_456
+        self.assertEqual(format(x, '_f'), '123_456.123456')
+        self.assertEqual(format(x, '._f'), '123456.123_456')
+        self.assertEqual(format(x, '_._f'), '123_456.123_456')
+        self.assertEqual(format(x, '.10_f'), '123456.123_456_000_0')
+        self.assertEqual(format(x, '>21._f'), '       123456.123_456')
+        self.assertEqual(format(x, '<21._f'), '123456.123_456       ')
```

Comment:
Maybe it would be better that way (I think that fine).
```
' ' * 7  + '123456.123_456'
'123456.123_456' + ' ' * 7
```

Reply:
I doubt.  Your version will be even slightly longer.

Hint: if you do several comments on changed files - it's better to push "Start a review", then add comments and finish with "Review changes".  In this way subscribed people got just one email notification.

Reply:
Oh, I'm sorry. After I finished reviewing it, it suddenly occurred to me that there was another place that might need to be explained.

Reply:
It's just a suggestion.